e2e/helpers/ACCESS_TYPE.ts

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
Duplicates synapse-type's [ACCESS_TYPE.ts](https://github.com/Sage-Bionetworks/synapse-web-monorepo/blob/main/packages/synapse-types/src/ACCESS_TYPE.ts) -- can be dropped when SWC-6529 is complete.